### PR TITLE
136 removing dry notes

### PR DESCRIPTION
**Pull request by ikopke23**
Check one:
- [ ] Semantic Change: something about the meaning of the text is different
- [x] Non-semantic Change: Spelling, grammar, or formatting changes.

Summary of change(s):

removes 'note' and 'additional note' from 5.D.2.B.1 and 5.D.2.B.2, (4.A.2.A.1 and .2)

those notes are now under 4.A.2.A

### CODE REVIEW

File: constitution.tex [hunk 1]
```diff
@@ -173,12 +176,7 @@
             A subset of Introductory Members will be offered On-Floor status and will be able to select a room on floor. 
             The other Introductory Members will have Off-Floor status, but will otherwise have the same privileges and responsibilities.
 \end{enumerate}
-Note: Most membership privileges do not initiate until successful completion of the introductory process.
```
Comment (danisaba) [inline constitution.tex:176]: why were there two :sob:

File: constitution.tex [hunk 2]
```diff
@@ -205,17 +203,6 @@
 \end{itemize}
 At the discretion of the Evaluations Director, any Introductory Packet may be extended to accommodate extenuating circumstances.
 \asubsubsection{Expectations of an Introductory Member}
-Before the end of the Process, an Introductory Member is expected to:
```
Comment (danisaba) [inline constitution.tex:208]: what happened here?

File: constitution.tex [hunk 3]
```diff
@@ -992,10 +979,21 @@ \subsubsection{Selection of the Chair, Evals Director, Social Director(s), Finan
 There are two methods for non-semantic change to the Constitution.
 A Maintainer may approve any proposed change that does not affect the meaning of the document.
 Alternatively, the change may be presented to the House for discussion followed by an Immediate Vote.
-A quorum of fifty percent of the Total Number of Possible Votes is required for passage.
-
-\asubsection{Semantic Changes}
-Any semantic change to the Constitution requires the change to be proposed in writing for discussion at a House Meeting.
+Any modifications made due to the discussion are added to the written proposal and the modified proposal is posted in the House during the week.
```
Comment (danisaba) [inline constitution.tex:982]: i am also not sure what happened here

**Review by danisaba — CHANGES_REQUESTED** [review]

**Comment by ikopke23** [comment]
saying what my recent commit said but I nuked my constitution.tex and restarted. 

All my skill issues that I assume happened as I was still learning how the fuck nvim worked have disappeared. The only difference should now pertain to the initial PR.

**Review by danisaba — APPROVED** [review]
Looks good to me.

**Review by esoccoli — APPROVED** [review]
Lgtm

**Pull request merged**